### PR TITLE
Associate pipeline with operator

### PR DESCRIPTION
**Pull request by aef-**

### CODE REVIEW

File: beagle/settings.py [hunk 1]
```diff
@@ -278,6 +278,7 @@
 
 POOLED_NORMAL_FILE_GROUP = os.environ.get('BEAGLE_POOLED_NORMAL_FILE_GROUP', '1552617b-3b06-46de-921a-a000a9129385')
 
+CELERY_TASK_ALWAYS_EAGER=DEBUG
```
Comment (stevekm) [inline beagle/settings.py]: @aef- How does this work? What behavior does it introduce? sounds useful

**Comment by stevekm** [comment]
hey can you try merging the latest develop into this branch and check if the tests work?

File: beagle/settings.py (hunk 1 above)
Comment (aef-) [reply beagle/settings.py]: I updated it to use Celery 4 settings but it's exactly what you did in the test files.

**Comment by aef-** [comment]
@stevekm updated so tests passed, thanks for writing them btw, they really helped out.

We may want to look at using pytest and factoryboy in the future when fixtures become a pain to manage due to model changes

File: lib/admin.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+def link_relation(field_name):
+    """
+    Converts a foreign key value into clickable links.
+
+    If field_name is 'parent', link text will be str(obj.parent)
+    Link will be admin url for the admin url for obj.parent.id:change
+    """
+    def _link_relation(obj):
+        print(field_name)
```
Comment (aef-) [inline lib/admin.py]: will remove.

**Pull request merged**